Content.Server/_ES/Masks/Objectives/Relays/Components/ESDamageTakerRelayComponent.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Content.Server._ES.Masks.Objectives.Relays.Components;
+
+[RegisterComponent]
+[Access(typeof(ESDamageTakerRelaySystem))]
+public sealed partial class ESDamageTakerRelayComponent : Component
+{
+
+}
```

Comment:
```suggestion
public sealed partial class ESDamageTakerRelayComponent : Component;
```
can just do this tbh